test/performance-charts/tests/ScatterChart.bench.tsx

```diff
@@ -24,7 +24,7 @@ describe('ScatterChart', () => {
     async () => {
       const { findByText } = render(
         <ScatterChart
-          xAxis={[{ data: xData }]}
+          xAxis={[{ data: xData, valueFormatter: (v) => v.toLocaleString('en-US') }]}
```

Comment:
I don't know why but I had an issue on my computer. The default locale of `.toLocaleString` was not the same in the test and in the render (`1 000` vs `1,000`)